src/test/groovy/org/jenkinsci/gradle/plugins/jpi/JpiPluginSpec.groovy

```diff
@@ -95,15 +96,20 @@ class JpiPluginSpec extends Specification {
         task != null
         task.description != null
         task.group == BasePlugin.BUILD_GROUP
-        task.archiveName == archiveName
-        task.extension == extension
+        if (GradleVersion.current() >= GradleVersion.version('5.0')) {
+            task.archiveFileName.get() == fileName
+            task.archiveExtension.get() == extension
+        } else {
+            task.archiveName == fileName
+            task.extension == extension
+        }
```

Comment:
I think I'd prefer to delete (rather than replace) these lines for two main reasons:

1. [JpiIntegrationSpec](https://github.com/jenkinsci/gradle-jpi-plugin/blob/master/src/test/groovy/org/jenkinsci/gradle/plugins/jpi/JpiIntegrationSpec.groovy#L26-L108) more comprehensively tests these extension properties.
2. The IntegrationSpec is using GradleTestKit, which automatically runs against checked-in Gradle and every version [we define in the buildscript](https://github.com/jenkinsci/gradle-jpi-plugin/blob/master/build.gradle.kts#L113). This spec is only ever run against checked-in Gradle.

After removing these lines we're just asserting the war and jpi tasks exist and have a description, so I'd also be OK with removing this test entirely.

Reply:
I stripped the test down to a minium. I think it's still useful to have a quick feedback about if the `jpi` and `war` tasks are configured.